worlds/tloz_oos/data/Constants.py

```diff
@@ -398,7 +398,7 @@
         'Sunken City: Syrup Trade',
         'Horon Village: Tick Tock Trade',
         'Eastern Suburbs: Guru-Guru Trade',
-        'Mt. Cucco: Chest Behind Talon',
+        'Mt. Cucco: Talon Trade',
```

Comment:
From the fact that syryp shop checks are here, Chest behind Talon should also be there (and Talon trade should be just after yelling old man as it's the vanilla order)